projects/aca-content/src/lib/components/sidenav/user-menu/user-menu.component.ts

```diff
@@ -50,9 +50,28 @@ export class UserMenuComponent implements OnInit {
   @Input()
   data: { items: any[] };
 
+  @ViewChild(MatMenu)
+  menu: MatMenu;
+
+  @ViewChildren(ToolbarMenuItemComponent)
+  toolbarMenuItems: QueryList<ToolbarMenuItemComponent>;
+
   ngOnInit() {
     if (this.data?.items) {
       this.data.items.sort((a, b) => a.order - b.order);
     }
   }
+
+  ngAfterViewInit() {
+    const menuItems: MatMenuItem[] = [];
+    this.toolbarMenuItems.forEach((toolbarMenuItem: ToolbarMenuItemComponent) => {
+      if (toolbarMenuItem.menuItem !== undefined) {
+        menuItems.push(toolbarMenuItem.menuItem);
+      }
+    });
+    const menuItemsQueryList: QueryList<MatMenuItem> = new QueryList<MatMenuItem>();
```

Comment:
no need to set type at the left side when we assigning inline

Reply:
done